packages/starknet/README.md

```diff
@@ -0,0 +1,113 @@
+# starknet.dart
+
+The goal of this SDK is to be able to interact with StarkNet smart contracts in a type-safe way.
+
+You can also call the JSON-RPC endpoint exposed by the StarkNet full nodes (see the [specs](https://github.com/starkware-libs/starknet-specs)).
+
+📚 [docs](https://pub.dev/packages/starknet)
+
+💬 [telegram chat](https://t.me/+CWezjfLIRYc0MDY0)
+
+## Motivation
+
+StarkNet is a revolution in the web3 world: it allows to [scale Ethereum](https://docs.ethhub.io/ethereum-roadmap/layer-2-scaling/zk-rollups/) and offers the best possible UX thanks to unique features like [account abstraction](https://www.argent.xyz/blog/wtf-is-account-abstraction/) or [session keys](https://github.com/dontpanicdao/starknet-burner).
+
+But web3 mainstream adoption won't happen unless decentralized applications go to mobile.
+
+That's why it's a priority to **build the best possible Starknet SDK for dart applications**, thus unlocking the era of Flutter mobile apps on StarkNet.
+
+## Roadmap
+
+You can follow the progress of the project directly on the [kanban](https://github.com/users/gabsn/projects/1).
+
+## Contributing
+
+If you want to contribute to this project or have any suggestion, please check out our [Contributor Guide](../../CONTRIBUTING.md).
+
+## Dev Guide
+
+### Initial Setup
```

Comment:
Initial setup need to be run in git repository top directory where [`pyproject.toml`](../../blob/main/pyproject.toml) is located

I suggest to add these commands in top directory README and refer this file here.